tests/test_model.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import pytest
+import peewee
+
+from tracboat.gitlab import model
+
+
+@pytest.mark.parametrize('version', [
+    '8.4',
+    '8.5',
+    '8.7',
+    '8.13',
+    '8.15',
+    '8.16',
+    '8.17',
+    '9.0.0'
```

Comment:
imho `9.0.0` model should be renamed to `9.0` like the rest of the models, to exclude patch revision.